fetch_description/robots/fetch.urdf

```diff
@@ -565,7 +565,7 @@
     <child link="laser_link" />
     <axis xyz="0 0 0" />
   </joint>
-  <link name="torso_fixed_link">
+  <link name="torsoed_link">
```

Comment:
Let's keep the original link/joint name of torso_fixed_joint, as well.

Reply:
Ok Fixed I think now. Made a replace to fast ;). 
Tell me if there is anything else and notify me if we merge